generative/maisi/configs/config_maisi.json

```diff
@@ -0,0 +1,166 @@
+{
+    "quality_check_args": null,
+    "num_output_samples": 1,
+    "body_region": ["abdomen"],
+    "anatomy_list": ["liver"],
+    "controllable_anatomy_size": [],
+    "num_inference_steps": 1000,
+    "mask_generation_num_inference_steps": 1000,
+    "random_seed": null,
+    "spatial_dims": 3,
+    "image_channels": 1,
+    "latent_channels": 4,
+    "output_size": [
+        256,
+        256,
+        256
+    ],
+    "output_ext": ".nii.gz",
+    "spacing": [
+        1.0,
+        1.0,
+        1.0
+    ],
+    "mask_generation_latent_shape": [
+        4,
+        64,
+        64,
+        64
+    ],
+    "autoencoder_def": {
+        "_target_": "autoencoderkl_maisi.AutoencoderKlMaisi",
+        "spatial_dims": "@spatial_dims",
+        "in_channels": "@image_channels",
+        "out_channels": "@image_channels",
+        "latent_channels": "@latent_channels",
+        "num_channels": [
+            64,
+            128,
+            256
+        ],
+        "num_res_blocks": [2,2,2],
+        "norm_num_groups": 32,
+        "norm_eps": 1e-06,
+        "attention_levels": [
+            false,
+            false,
+            false
+        ],
+        "with_encoder_nonlocal_attn": false,
+        "with_decoder_nonlocal_attn": false,
+        "use_checkpointing": false,
+        "use_convtranspose": false
+    },
+    "difusion_unet_def": {
+        "_target_": "custom_network_diffusion.CustomDiffusionModelUNet",
+        "spatial_dims": "@spatial_dims",
+        "in_channels": "@latent_channels",
+        "out_channels": "@latent_channels",
+        "num_channels": [
+            64,
+            128,
+            256,
+            512
+        ],
+        "attention_levels": [
+            false,
+            false,
+            true,
+            true
+        ],
+        "num_head_channels": [
+            0,
+            0,
+            32,
+            32
+        ],
+        "num_res_blocks": 2,
+        "use_flash_attention": true,
+        "input_top_region_index": true,
+        "input_bottom_region_index": true,
+        "input_spacing": true
+    },
+    "controlnet_def": {
+        "_target_": "custom_network_controlnet.CustomControlNet",
```

Comment:
We will refactor this to monai core once our PRs are merged, right?

Reply:
Yes, please change it to monai core after PR merged